examples/use-cases/push_events_to_syslog/pull_events.py

```diff
@@ -0,0 +1,83 @@
+import automox_console_sdk as automox
+import os
+import time
+import socket
+import json
+
+from automox_console_sdk import EventsApi
+from getpass import getpass
+
+def getRecentEvents(event_id=None):
+    try:
+        page = 0
+
+        while True:
+            recent_events = []
+            events_page = events_api.get_events(o=organization, limit=500, page=page)
```

Comment:
Could these calls be wrapped in an error check in case the API key is incorrect or another error was returned?

Reply:
There is a try..catch on line 11 that should capture any exceptions thrown by get_events(). The client initialization is also wrapped in one -- or am I missing something?

Reply:
🤦 disregard.